go.mod

```diff
@@ -9,34 +9,76 @@ replace (
 	gopkg.in/check.v1 => github.com/kastenhq/check v0.0.0-20180626002341-0264cfcea734
 )
 
+require (
+	github.com/Azure/azure-sdk-for-go v54.0.0+incompatible
```

Comment:
we see some of the un necessary changes here because I think my vs code grouped the dependencies in direct and indirect dependencies and I liked it this way.

Reply:
Should we add comments about direct and indirect so that in the future people add new dependencies in the right list?